stdlib/REPL/src/Terminals.jl

```diff
@@ -101,8 +101,13 @@ pipe_writer(t::UnixTerminal) = t.out_stream::IO
 
 mutable struct TerminalBuffer <: UnixTerminal
     out_stream::IO
+    buf::IOBuffer
 end
 
+TerminalBuffer(terminal::TextTerminal) = TerminalBuffer(terminal, IOBuffer())
```

Comment:
seems like this is writing to the wrong stream now, based on expected/past/current usage?
```suggestion
    out_stream::IOBuffer
    ctx::IO
end

TerminalBuffer(terminal::UnixTerminal) = TerminalBuffer(IOBuffer(), terminal)
```

(aside, seems like this type some slightly unnecessary alternative to using a `TTYTerminal(devnull, IOContext(IOBuffer(), terminal.out_stream), devnull)` and deleting `UnixTerminal` and `TextTerminal`, but eh, whatever, that is a pre-existing duplication)